VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -74,7 +74,7 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
         provider: { VideoStreamViewController() }
     )
     
-    private var observer: PlayerObserver?
+    private var observer: SystemPlayerObserver?
```

Comment:
We have observers for our `Player` and to separate it from `AVPlayer` this name is preferrable for me.